src/gc/gc.cpp

```diff
@@ -22544,7 +22544,7 @@ void gc_heap::make_free_lists (int condemned_gen_number)
 
 void gc_heap::make_free_list_in_brick (BYTE* tree, make_free_args* args)
 {
-    assert ((tree >= 0));
+    assert ((tree > 0));
```

Comment:
Since you are touching this, it seem it would be nicer to check for inequality with NULL instead.